tests/databases/api_tests.py

```diff
@@ -820,7 +820,21 @@ def test_test_connection_failed(self):
         self.assertEqual(rv.headers["Content-Type"], "application/json; charset=utf-8")
         response = json.loads(rv.data.decode("utf-8"))
         expected_response = {
-            "message": "Could not load database driver: BaseEngineSpec",
+            "errors": [
+                {
+                    "message": "Could not load database driver: BaseEngineSpec",
```

Comment:
i guess ideally in the future these messages will become actual errors with details on how to resolve them in the documentation

Reply:
Right, we're going to work on all the messaging in the test connection flow and add database flow.